Automattic/simplenote-ios
# Add lanes for automatic build and upload

This PR adds a few lanes to automate the build and the upload of the binaries to TestFlight and HockeyApp. 

# To Test:
1. Make sure to add the new required entries to your `~/.simplenoteios-env.default`.
2. Checkout a new branch.
3. change `VERSION_LONG` in `Version.Public.xcconfig` to a new version that has been uploaded yet. The best thing during testing is to change only the last number because it's the one that's not seen by the users.
4. Commit the changes.  
5. Run `bundle install`.
6. Run `bundle exec fastlane build_and_upload_release`.
7. Verify that the system reads the correct version and confirm.
8. Verify that the builds run without errors.
9. When prompted, login into TestFlight.
10. Verify that the process finishes without errors.
11. Verify that the build has been uploaded to HockeyApp.
12. Verify that the build has been uploaded to TestFlight.
